doc/source/multiscene.rst

```diff
@@ -231,3 +231,46 @@ multiple Scenes use:
     >>> mscn = MultiScene.from_files(glob('/data/abi/day_1/*C0[12]*.nc'), reader='abi_l1b')
     >>> mscn.load(['C01', 'C02'])
     >>> mscn.save_datasets(base_dir='/path/for/output')
+
+Combining multiple readers
+--------------------------
+
+Since Satpy 0.23, it is possible to automatically
```

Comment:
Would it be possible to use the `.. versionadded::` directive at the top or bottom of this section instead of mentioning it here. It makes the instructions cleaner in my opinion.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

Reply:
Replaced by `.. versionadded::`.